checkout/features/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Features
```

Comment:
We need to add `redirect_from: /<documentation section>/other-features` in all `features/index.md` files so people with the old URL bookmarked is directed to the new location.

```suggestion
title: Features
redirect_from: /checkout/other-features
```